src/App.tsx

```diff
@@ -1,25 +1,116 @@
 import './App.scss';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUser(userId: number): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+export const App: React.FC = () => {
+  const [title, setTitle] = useState('');
+  const [visibilityError, setVisibilityTheError] = useState(true);
```

Comment:
```suggestion
  const [isError, setIsError] = useState(true);
```
![image](https://user-images.githubusercontent.com/96409563/224259138-6b609c0e-3d94-42a5-8158-79becf1e1781.png)